src/pages/CreateDesign/Pattern/types.ts

```diff
@@ -1 +1,2 @@
 export const defaultFontSize = 16;
```

Comment:
p3: 얘는 상수인데 카멜케이스이네요! 
```suggestion
export const DEFAULT_FONT_SIZE = 16;
```
이렇게 수정하는건 어떠신가요?